[Wearable] Fix selector icons disappearing

ampanasiuk opened the pull request:
[Issue] https://github.com/Samsung/TAU/issues/369
[Problem] updating style.transform with single-arg scale() value in js
          makes the decimal point replaced with comma, at least on
          some locales
[Solution] use two-arg style.transform scale() value

ampanasiuk commented:
This works in that icons don't disappear, but sadly retrieving style.transform has "scale(0,8235, 0,8235)" and this doesn't work with "newTransformStyle = transform.replace(DEFAULT.ITEM_NORMAL_SCALE, DEFAULT.ITEM_ACTIVE_SCALE);", so the active item's size is not increased.

ampanasiuk commented:
New approach

TomaszLukawskiSam reviewed (changes requested):
Hi, These changes you should do in `src/js/profile/wearable/widget/wearable/Selector.js`
The `dist` folder is replaced during build process.

ampanasiuk committed:
[Wearable] Fix selector icons disappearing

[Issue] https://github.com/Samsung/TAU/issues/369
[Problem] retrieving style.transform on some locales has the decimal
          point in the scale() value replaced with comma
[Solution] do not depend on the decimal point or comma being present

TomaszLukawskiSam reviewed (changes requested):
Hi, would you change the PR to branch `tau_1.0`?
